cfme/tests/ansible/test_embedded_ansible_crud.py

```diff
@@ -133,3 +99,30 @@ def test_embedded_ansible_logs(embedded_appliance):
     # Retrieving setup log file from list and asserting with length
     # Setup log file contains date/time string in it.
     assert "setup" in diff[0]
+
+
+@pytest.mark.tier(3)
+def test_embedded_ansible_disable(embedded_appliance):
```

Comment:
@sbulage I am suspect its order problem. I think better solution of this to add small fixture like `ensure_enabled` or `ensure_disable` which will reduce your test dependency.

Reply:
Okay @digitronik I will update it.

Reply:
Updated.